tests/parser.cpp

```diff
@@ -36,7 +36,7 @@ void test_parse_failure(std::string_view input,
 void test(BPFtrace &bpftrace, std::string_view input, std::string_view expected)
 {
   Driver driver(bpftrace);
-  EXPECT_EQ(driver.parse_str(input), 0);
+  ASSERT_EQ(driver.parse_str(input), 0);
```

Comment:
If this fails then `driver.root` will be a null pointer and `Printer` will crash below trying to display it